pkg/scrape/vrbangers.go

```diff
@@ -92,7 +92,7 @@ func VRBangersSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, ou
 			sc.Covers = append(sc.Covers, e.ChildAttrs(`section.banner picture img`, "data-pagespeed-lazy-src")[0])
 		}
 
-		if len(e.ChildAttrs(`section.base-content--bg img`, "data-pagespeed-lazy-src")) > 0 {
+		if len(e.ChildAttrs(`section.base-content--bg div.base-border img`, "data-pagespeed-lazy-src")) > 0 {
 			sc.Covers = append(sc.Covers, e.ChildAttrs(`section.base-content--bg img`, "data-pagespeed-lazy-src")[0])
 		}
```

Comment:
The same selector should probably be used when retrieving the cover:
```suggestion
		if len(e.ChildAttrs(`section.base-content--bg div.base-border img`, "data-pagespeed-lazy-src")) > 0 {
			sc.Covers = append(sc.Covers, e.ChildAttrs(`section.base-content--bg div.base-border img`, "data-pagespeed-lazy-src")[0])
		}
```
Or better yet avoid repeating it with something like this:
```suggestion
		if tmpCovers := e.ChildAttrs(`section.base-content--bg div.base-border img`, "data-pagespeed-lazy-src"); len(tmpCovers) > 0 {
			sc.Covers = append(sc.Covers, tmpCovers[0])
		}
```

Reply:
Thanks, your suggestions actually uncovered some inconsistency in the web code, so I made additional change..